marshmallow/validate.py

```diff
@@ -271,6 +274,48 @@ def __call__(self, value):
         return value
 
 
+class OneOf(object):
+    """Validator which succeeds if ``value`` is a member of ``choices``.
+
+    :param iterable choices:
```

Comment:
For consistency with the rest of the codebase, don't add an extra newline after `param_name:`.